packages/gestalt-charts/src/ChartGraph.js

```diff
@@ -0,0 +1,586 @@
+                  tooltip={tag.tooltip}
+                />
+              ))}
+            </Flex>
+          </Box>
+        ) : null}
+        <div className="_gestalt" style={{ width: '100%', height: '100%', maxWidth: 960 }}>
+          <ResponsiveContainer
+            debounce={150}
+            onResize={(width, height) => {
+              setChartHeight(height);
+              setChartWidth(width);
+            }}
+            minWidth="100%"
+            width="100%"
+            minHeight={internalHeight}
+            height={internalHeight}
+          >
+            <ChartType
+              title={`${accessibilityLabelPrefixText}. ${accessibilityLabel}`}
+              {...(isBar || isComposed ? { barCategoryGap: '25%' } : {})}
+              data={data}
+              layout={isVertical ? 'vertical' : 'horizontal'}
+              margin={{
+                top: 10,
+                right: 5,
+                bottom: isVerticalBiaxial && legend === 'auto' ? 20 : 10,
+                left: 5,
+              }}
+            >
+              {patterns}
+              <CartesianGrid
+                stroke="var(--color-border-container)"
+                horizontal={isVertical ? false : undefined}
+                vertical={isVertical ? undefined : false}
+              />
+              {isHorizontal ? (
+                <Fragment>
+                  <XAxis
+                    padding={
+                      isTimeSeries && (isBar || isComposed) ? { left: 100, right: 100 } : undefined
+                    }
+                    axisLine={false}
+                    dataKey="name"
+                    domain={isTimeSeries ? !Array.isArray(range) && range?.xAxisBottom : undefined}
+                    orientation="bottom"
+                    {...(isTimeSeries ? { scale: 'time' } : {})}
+                    style={FONT_STYLE_CATEGORIES}
+                    tickFormatter={
+                      isTimeSeries
+                        ? tickFormatter?.xAxisBottom || tickFormatter?.timeseries
+                        : (value: string) => labelMap?.[value] || value
+                    }
+                    tickLine={false}
+                    type={isTimeSeries ? 'number' : 'category'}
+                    // DO NOT SET xAxisId here
+                  />
+                  <YAxis
+                    axisLine={false}
+                    domain={Array.isArray(range) ? range : range.yAxisLeft}
+                    orientation="left"
+                    style={FONT_STYLE_VALUES}
+                    tickLine={false}
+                    tickCount={tickCount}
+                    yAxisId="left"
+                    tickFormatter={tickFormatter?.yAxisLeft}
+                  />
+                </Fragment>
+              ) : null}
+              {isHorizontalBiaxial ? (
+                <YAxis
+                  axisLine={false}
+                  domain={Array.isArray(range) ? range : range.yAxisLeft}
+                  orientation="right"
```

Comment:
Can we write a function that takes in the layout and spits out the axis. Looks like this is super similar to the top, and just two prop changes right? 

If we're cool with prop spreading, then just tweaking the object

It can probably make this a tad more compact
```
getYAxis(layout:horizontal|horizontalBiAxial|...){

}
```